src/LandIce/evaluators/LandIce_BasalFrictionCoefficient_Def.hpp

```diff
@@ -373,15 +449,15 @@ evaluateFields (typename Traits::EvalData workset)
   if (memoizer.have_saved_data(workset,this->evaluatedFields()))
     return;
 
-  if (beta_type == POWER_LAW) {
+  if ((beta_type == POWER_LAW)||(beta_type==REGULARIZED_COULOMB) ) {
```

Comment:
You might just add a `|| beta_type==GIVEN_CONSTANT` in the previous if statement, and keep the code below less indented.